dataland-frontend/src/utils/DataTableDisplay.ts

```diff
@@ -1,12 +1,23 @@
+
 /**
  * Sorts dates to ensure that Sfdr and LkSG datasets are displayed chronologically in the table in terms of reporting
  * periods (strings starting with numbers should at least be listed before those that do not)
  * @param  listOfDataDateToDisplayAsColumns list of objects to sort
+ * @param boolSwap toogles the swap of both list elements in listOfDataDateToDisplayAsColumns (in case there are two.
+ * Shortens the test-function and avoids code duplications.
  * @returns []
  */
 export function sortReportingPeriodsToDisplayAsColumns(
-  listOfDataDateToDisplayAsColumns: ReportingPeriodOfDataSetWithId[]
+  listOfDataDateToDisplayAsColumns: ReportingPeriodOfDataSetWithId[],
+  boolSwap = false
```

Comment:
Don't put this swap mechanic into this function but into a different test function. This doesn't obscure the code here and solves the code duplication issue in the test.